sql/slow-query.md

```diff
@@ -51,7 +51,7 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
 * `Is_internal`：表示是否是 TiDB 内部 SQL。true 为TiDB 内部执行的SQL, 比如 analyze，load variable 等；false 为用户执行的 SQL 。
 * `Digest`：表示 SQL 语句的指纹。
 * `Memory_max`：表示执行期间做多时候使用的内存数量, 单位为byte。
-* `Num_cop_tasks`：表示 cop-tasks 的数目。
+* `Num_cop_tasks`：表示 [cop-tasks](https://github.com/pingcap/docs-cn/blob/master/sql/understanding-the-query-execution-plan.md) 的数目。
```

Comment:
should we use relative links? @lilin90 PTAL

Reply:
@zz-jason Yes!